doc/localization.md

```diff
@@ -92,3 +92,23 @@ For example, if you want to replace `Chat` with `Notes`, simply add...
 , "pad.chat": "Notes"
 }
 ```
+
+## Customization for Administrators
+
+As an Etherpad administrator, it is possible to overwrite core mesages as well as messages in plugins. These include error messages, labels, and user instructions. Whereas the localization in the source code is in separate files separated by locale, an administrator's custom localizations are in `settings.json` under the `customLocaleStrigns` key, with each locale separated by a sub-key underneath.
+
+For example, let's say you want to change the text on the "New Pad" button on Etherpad's home page. If you look in `locales/en.json` (or `locales/en-gb.json`) you'll see the key for this text is `"index.newPad"`. You could add the following to `settings.json`:
+
+```
+  "customLocaleStrings": {
+    "fr": {
+      "index.newPad": "Créer un document"
```

Comment:
Chances are Google Translate gave me the infinitive instead of the imperative, so I probably am embarrassing myself here. I welcome any francophones to correct me.